docs/control-structures.rst

```diff
@@ -326,7 +352,7 @@ until the end of a ``{ }``-block. As an exception to this rule, variables declar
 initialization part of a for-loop are only visible until the end of the for-loop.
 
 Variables and other items declared outside of a code block, for example functions, contracts,
-user-defined types, etc., do not change their scoping behaviour. This means you can
+user-defined types, etc., do not change their scoping behavior. This means you can
```

Comment:
Can we make a decision on British vs. American? The documentation uses both, albeit I think historically it has more of the British spelling.

Reply:
@axic Yes, we have, with British, but pretty much all of the FAQ was US, so I changed as I went, this one snuck in… And over time I will correct the inconsistencies.